network_wrangler/Scenario.py

```diff
@@ -2,72 +2,129 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import annotations
-
-import os
-
+import os, sys
 from .ProjectCard import ProjectCard
+from collections import OrderedDict
+from .Logger import WranglerLogger
 
 class Scenario(object):
     '''
     Holds information about a scenario
     '''
 
-    def __init__(self, base_scenario: dict, project_cards: [ProjectCard] = None):
+    def __init__(self, base_scenario: dict, project_cards: [ProjectCard] = None, prerequisite_dict: dict = None, corequisite_dict: dict = None, conflict_dict: dict = None):
         '''
         Constructor
 
         args:
         base_scenario: the base scenario
-        project_card: this scenario's project card
+        project_cards: this scenario's project cards
         '''
 
         self.base_scenario = base_scenario
         self.project_cards = project_cards
 
-        self.prerequisites = {} # dictionary of
-        self.corequisites  = {}
-        self.conflicts     = {}
+        self.prerequisites = prerequisite_dict
+        self.corequisites  = corequisite_dict
+        self.conflicts     = conflict_dict
 
     @staticmethod
     def create_scenario(base_scenario: dict, card_directory: str = '', tags: [str] = None, project_cards_list = []) -> Scenario:
         '''
-        Validates project cards with a specific tag from the specified folder and
+        Validates project cards with a specific tag from the specified folder or
+        list of user specified project cards and
         creates a scenario object with the valid project card.
 
         args:
         base_scenario: the base scenario
         tags: only project cards with these tags will be read/validated
         folder: the folder location where the project cards will be
+        project_cards_list: list of project cards to be applied
         '''
-        scenario = Scenario(base_scenario, project_cards = project_card_list)
+
+        prereq_dict   = {}
+        coreq_dict    = {}
+        conflict_dict = {}
+
+        for project_card in project_cards_list:
```

Comment:
I think this part happens in add_project_cards()

Reply:
And I think add_project_cards() is called in __init__()

Reply:
`add_project_cards` is to bulk load the projects from a directory to a scenario object (self). I don't think this will be called from init(). But I have added part of its functionality to the init() too. Take a look at the latest commit and let me know.